content/applications/general/calendars/outlook/outlook_calendar.rst

```diff
@@ -1,103 +1,149 @@
+optional.
 
-.. note::
-   Regarding credentials, you *must* **Add a client secret**, called *Client Secret* in Odoo, which
-   allows Odoo to authenticate itself, requiring no interaction from your side. *Certificates* are
-   optional.
+Since resetting the synchronization can be tricky, Odoo recommends setting the maximum allowed
+:guilabel:`expiration date` for the Client Secret (24 months), so there is no need to
+re-synchronize soon. 
 
 Configuration in Odoo
 =====================
 
-Go to general *Settings* and activate the *Outlook Calendar* feature.
+In the Odoo database, go to :menuselection:`Settings --> General Settings` and activate the
+:guilabel:`Outlook Calendar` feature.
 
 .. image:: outlook_calendar/outlook_calendar.png
    :align: center
-   :alt: Outlook Calendar feature activated in Odoo
+   :alt: Outlook Calendar feature activated in Odoo.
 
-From your Azure portal, under *Overview*, copy your *Application (client) ID*, and paste it to
-*Client ID* in Odoo.
+From the Microsoft Azure portal, under the :guilabel:`Overview` section, copy the
+:guilabel:`Application (Client) ID`, and paste it into the :guilabel:`Client ID` field in Odoo.
 
 .. image:: outlook_calendar/application_client_ID.png
    :align: center
-   :alt: Client ID token to be copied from Microsoft to Odoo
+   :alt: Client ID token to be copied from Microsoft to Odoo.
 
-Going back to your Azure portal, now under *Certificates & secrets*, copy your *Client Secret Value*
-and paste it to *Client Secret* in Odoo.
+In the Microsoft Azure portal, under the :guilabel:`Certificates & secrets` section, copy the
+:guilabel:`Client Secret Value` and paste it into the :guilabel:`Client Secret` field in Odoo.
 
 .. image:: outlook_calendar/client_secrets.png
    :align: center
-   :alt: Client Secret token to be copied from Microsoft to Odoo
+   :alt: Client Secret token to be copied from Microsoft to Odoo.
 
 .. _outlook_calendar/sync_with_outlook:
 
 Sync with Outlook
 =================
 
-.. important::
-   | Odoo Calendar sends a confirmation email at the creation of an event to its attendees. But, on
-     the **first** sync, Outlook will also send confirmation emails of all scheduled events
-     (including past ones).
-   | To avoid this situation, we recommend *archiving all past events, and removing yourself from
-     future ones in Odoo Calendar before the first sync*. This will prevent events from being
-     created in Outlook, and therefore, no redundant emails will be sent during the first
-     synchronization. After the sync, events will only send one confirmation email.
+Warnings (before sync)
+----------------------
+
+Odoo highly recommends testing the Outlook calendar synchronization on a test database and a test
+email address (that is not used for any other purpose) before attempting to sync the desired
+Outlook Calendar with the user’s production database.
```

Comment:
This section starting at line 85 and below looks like it got copied in from a doc which uses a serif'd apostrophe instead of the sans-serif version we'd prefer to use in RST.

```suggestion
Outlook Calendar with the user's production database.
```